u_public.go

```diff
@@ -180,8 +254,13 @@ func (crm *certificateRequestMsgTLS13) toPublic() *CertificateRequestMsgTLS13 {
 	if crm == nil {
 		return nil
 	} else {
+		rawBytes := []byte{}
+		if raw, err := crm.marshal(); err != nil {
```

Comment:
Should we do something with this err? It looks like here we're just silently failing to include the rawBytes if the crm.marshal() fails. Are there errors we might have to catch here?

Reply:
The point of generating this rawBytes is to add it to the now deprecated `Raw` field for backwards compatibility (see prev comment). This raw field is not used in utls, so I think we might be able to get away with ignoring this error (and only providing the raw field when there is no error).